### PR TITLE
toString leveraged to json

### PR DESCRIPTION
**Pull request by eraly**
Works. Feedback requested.

### CODE REVIEW

File: nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ndarray/NdArrayJSONReader.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        int[] theShape = new int[rank];
+        int rows = 1;
+        for (int i = 0; i < rank; ++i) {
+            theShape[i] = objArr.optInt(i);
+            if (i != objArr.length() - 1)
+                rows *= theShape[i];
+        }
+        */
+        System.out.println("API_Error: Current support only for files written from dl4j");
+        return null;
```
Comment (agibsonccc) [inline nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ndarray/NdArrayJSONReader.java]: Loggers.

**Comment by agibsonccc** [comment]
Could you look at the test failures there? Maybe you need to update master on this branch?

**Comment by eraly** [comment]
Passes locally. Maybe a Jenkins thing.

**Comment by eraly** [comment]
Start Jenkins

**Comment by eraly** [comment]
Jenkins doesn't build. Debugging.

**Pull request closed without merging**